Misc/NEWS.d/next/Library/2020-02-16-07-47-55.bpo-27657.9kZchc.rst

```diff
@@ -0,0 +1,4 @@
+In "bpo-27657: Fix urlparse() with numeric paths (GH-661) (#16839)", the
+change broke the backwards compatibility of an undefined parsing behavior in
+a patch release. The behavior was relied upon by users. A decision was taken
```

Comment:
I think we should explicitly mention the releases involved.  For 3.7, how about something like:

> The original fix for bpo-27657, "Fix urlparse() with numeric paths" (GH-16839)
> included in 3.7.6, inadvertently introduced a behavior change that broke several third-party packages relying on the original undefined parsing behavior.  The change is reverted in 3.7.7, restoring the behavior of 3.7.5 and earlier releases.

Reply:
Thanks, Ned. I have adopted this suggestion.